libs/ui-lib/lib/ocm/api/axiosClient.ts

```diff
@@ -15,22 +15,25 @@ const axiosCaseConverterOptions = {
   },
 };
 
-const getDefaultClient = (withoutConverter = false) => {
+const token = '';
```

Comment:
Danger zone!
Is this really necessary?

Reply:
I'll revert the file and only add the `isInOcm` to avoid passing the entire `ocmClient` in many different components.